### ISSUE TITLE
NPM org style package names. (@thing/stuff)

### ISSUE THREAD

**Issue by digitalsadhu** [eik-lib/core#72]
I've added uri encoding to handle this on the cli end but still having issues.

Eg.
```
join(this.org, 'pkg', encodeURIComponent(this.name), this.version)
```

Server is responding that version is not valid

**Comment by trygve-lie** [comment]
Funny. We are using npm's own module for verify npm module names. I'll take a look..

**Comment by trygve-lie** [comment]
The package name should be supported. That is tested here: https://github.com/eik-lib/common/blob/master/test/validators/index.js#L84-L85. Weird that its version which seems to be wrong.

**Comment by trygve-lie** [comment]
Been digging around this and this is a little bit bigger change to get support for but I think its very important to get this in there. I've got a WIP on #90 for this.

**Comment by trygve-lie** [comment]
Fixed in #90
